test/server/cards/01_SOR/units/RuthlessRaider.spec.ts

```diff
@@ -0,0 +1,49 @@
+describe('Ruthless Raider', function() {
+    integration(function(contextRef) {
+        describe('Ruthless Raider\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['inferno-four#unforgetting', 'system-patrol-craft'],
+                        hand: ['ruthless-raider']
+                    },
+                    player2: {
+                        spaceArena: ['green-squadron-awing'],
+                        groundArena: ['wampa'],
+                        hand: ['rivals-fall']
+                    }
+                });
+            });
+
+            it('should deal 2 damage to an enemy unit and enemy base.', function () {
+                const { context } = contextRef;
+
+                // Play Ruthless Raider from hand
+                context.player1.clickCard(context.ruthlessRaider);
+
+                // Select Enemy Unit and Base. Not able to select friendly units
+                expect(context.player1).toBeAbleToSelectExactly([context.greenSquadronAwing, context.wampa, context.player2.base]);
+                context.player1.clickCard(context.greenSquadronAwing);
+                context.player1.clickCard(context.p2Base);
```

Comment:
This click shouldn't be here, the base will be automatically selected for `theirBase` since it's the only legal target that fits the filters